app/Makefile

```diff
@@ -1,15 +1,21 @@
+all:
+	@echo "run:			meteor npm start"
+	@echo "parse-server:		meteor npm run server"
+	@echo "run-test:		meteor npm run test"
+	@echo "test-watch:		meteor npm run test-watch"
+
 # Dev server
 run:
-	npm run start
+	meteor npm start
```

Comment:
Looks like `parse-server` is the only command that needs "run". 
I've updated accordingly. @sturmer-eha can you confirm?